tests/toolchain/crash.pm

```diff
@@ -44,7 +44,11 @@ sub run() {
         my $url = "ftp://openqa.suse.de/" . get_var('REPO_SLES_DEBUG');
         assert_script_run "zypper ar -f $url SLES-Server-Debug";
 
-        assert_script_run 'zypper ref; zypper -n -v in kernel-default-base-debuginfo kernel-default-debuginfo', 300;
+        # workaround before POOL metadata in Media3 are fixed
```

Comment:
don't forget to record a soft failure with a reference to a bug for every "product workaround" we have in code.